OpenRA.Platforms.Default/Sdl2GraphicsDevice.cs

```diff
@@ -98,6 +86,47 @@ public Sdl2GraphicsDevice(Size windowSize, WindowMode windowMode)
 
 			SDL.SDL_SetModState(SDL.SDL_Keymod.KMOD_NONE);
 			input = new Sdl2Input();
+
+			Clear();
+			Present();
+			SetWindowSize(WindowSize, windowMode);
+		}
+
+		public void SetWindowSize(Size windowSize, WindowMode windowMode)
```

Comment:
You're now handling the window mode as well, so calling this `SetWindowSize` doesn't make sense.  It should be changed to `SetWindowMode`.